src/EditorFeatures/Core/Implementation/IntelliSense/Completion/FileSystem/FileSystemCompletionHelper.cs

```diff
@@ -228,9 +228,13 @@ private bool ShouldShow(FileSystemInfo child)
 
         private bool CanAccess(FileSystemInfo info)
         {
-            return info.TypeSwitch(
-                (DirectoryInfo d) => CanAccessDirectory(d),
-                (FileInfo f) => CanAccessFile(f));
+            switch (info)
+            {
+                case DirectoryInfo d: return CanAccessDirectory(d);
```

Comment:
I welcome some sort of formatter that enforces those rules.  The IDE is certainly full of single line switch statements :)